test/Feature/TypedBuffer/GetDimensions.test

```diff
@@ -0,0 +1,91 @@
+#--- source.hlsl
+
+// This test checks that we will get the expected values from invoking
+// `GetDimension`s on typed buffers RWBuffer and Buffer.
+
+Buffer<int4> A : register(t0);
+RWBuffer<float> B : register(u1);
+
+[[vk::binding(10)]]
+RWBuffer<uint> Out : register(u10);
+
+[numthreads(4,1,1)]
```

Comment:
Feature doesn't take advantage of any threading and you aren't using a groupid index so no reason for this to be 4. 
```suggestion
[numthreads(1,1,1)]
```